app/scripts/controllers/membership.js

```diff
@@ -88,16 +91,16 @@ angular
           .then(function() {
             resetForm();
             refreshRoleBindingList();
-            showAlert('rolebindingCreate', 'success', messages.update.subject.success({
+            showAlert('rolebindingCreate', 'success', escaped.update.subject.success({
               roleName: role.metadata.name,
               subjectName: _.escape(newSubject.name)
```

Comment:
Just to make sure I'm with you, alerts still get user input text, I'm just removing the `_.escape()` in favor of using `<%- ` in the templates, separating out the one template that processes server response with `<%=` (I could see this one being inline but would rather be consistent at this point).